src/components/Banner/Banner.stories.jsx

```diff
@@ -19,6 +19,7 @@ const Template = args => <Banner {...args} />;
 const defaultArgs = {
   visible: true,
   title: 'Coronavirus',
+  storage: { getItem: () => {}, setItem: () => {} },
```

Comment:
The Banner is designed not to close without this prop :shrug: 

https://github.com/department-of-veterans-affairs/component-library/blob/07dd499dbdb5c6b5f13b03977167f019adbd2bf6/src/components/Banner/Banner.jsx#L130-L131